### PR TITLE
V8 upgrade

### PR DESCRIPTION
**Pull request by dasanchez**
- Renamed the testnet folders
- Updated info on the Theta public and local testnet pages.
- Upgrade dates still TBD

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,40 +2,36 @@
...
 -  Provide developers with a low-risk production-like environment to test integrations with Cosmos Hub.
 -  Make historical hub software releases and states readily available for testing and querying.
-  
+
 Beyond these goals, testnets could also become a site for R&D for new development and governance approaches in a fast-moving and live context.
 
-We are expecting the roll out as follows:
+Up until the `Vega` testnet, our approach was to deploy a testnet for each Gaia upgrade.
+
+**Starting with `Theta`, we have moved to a persistent testnet model, which means the `Theta` testnet will stay online and remain the primary Cosmos Hub testnet after the `v7-Theta` upgrade and beyond.**
```
Comment (uditvira) [inline README.md]:
```suggestion
**Starting with `Theta`, we have moved to a persistent testnet model. The`Theta` testnet will stay online and remain the primary Cosmos Hub testnet after the `v7-Theta` upgrade, including for the v8-Rho and v9-Lambda upgrades, and beyond**
```

File: theta/public-testnet/UPGRADES.md [hunk 2]
```diff
@@ -1,16 +1,35 @@
...
+| October 2022  |   Voting ends                 |
+| October 2022  |   Rho upgrade ([Gaia v8.0.0-rc0](https://github.com/cosmos/gaia/releases/tag/v8.0.0-rc0)) is live on the testnet |
+
+
+## 2022-10 v8-Rho upgrade
+
+* **Version before upgrade**: `v7.0.x`
+* **Version after upgrade**: `v8.0.0-rc0`
+
+
```
Comment (uditvira) [inline theta/public-testnet/UPGRADES.md]: remove extra lines?

**Review by uditvira — CHANGES_REQUESTED** [review]

**Comment by uditvira** [comment]
@ASoTNetworks can you update the devnet information?

**Comment by dasanchez** [comment]
Closing this PR to open one against `cosmos/testnets`

**Pull request closed without merging**